src/libdredd/src/mutate_visitor.cc

```diff
@@ -278,6 +279,29 @@ bool MutateVisitor::VisitBinaryOperator(
   return true;
 }
 
+bool MutateVisitor::VisitExpr(clang::Expr* expr) {
+  if (!IsInFunction()) {
+    // Only consider mutating expressions that occur inside functions.
+    return true;
+  }
+
+  // For unary and binary operators, unary operator insertion and
+  // statement replacement are handled in these classes.
```

Comment:
I wouldn't say "in these classes" - not clear from the context what those refer to. How about saying that unary and binary operators are intercepted separately?

Reply:
Changed.